### PR TITLE
Explicit objcopy definition

### PR DESCRIPTION
**Pull request by cincuranet**
Makes sure to use proper one when building the NativeAOT app.

### CODE REVIEW

**Comment by cincuranet** [comment]
cc @eerhardt

**Review by LoopedBard3 — COMMENTED** [review]
Could you expand on what the additional '/p:ObjCopyName=objcopy' does? Taking a quick look online did not come up with anything specific.

**Comment by cincuranet** [comment]
It's described here: https://github.com/dotnet/runtime/pull/81517#issuecomment-1413517696.

**Review by LoopedBard3 — APPROVED** [review]
LGTM

File: src/scenarios/emptyconsolenativeaot/pre.py [hunk 1]
```diff
@@ -13,7 +14,8 @@
                 bin_dir=const.BINDIR,
                 exename=EXENAME,
                 working_directory=sys.path[0])
-precommands.execute(['/p:PublishAot=true', '/p:StripSymbols=true', '/p:IlcGenerateMstatFile=true'])
+args = ['/p:PublishAot=true', '/p:StripSymbols=true', '/p:IlcGenerateMstatFile=true'] if iswin() else ['/p:PublishAot=true', '/p:StripSymbols=true', '/p:IlcGenerateMstatFile=true', '/p:ObjCopyName=objcopy']
```
Comment (eerhardt) [inline src/scenarios/emptyconsolenativeaot/pre.py:17]:
```py
args = ['/p:PublishAot=true', '/p:StripSymbols=true', '/p:IlcGenerateMstatFile=true']
if not iswin():
	args.append('/p:ObjCopyName=objcopy')
```

**Pull request merged**